code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -53,3 +53,8 @@
 	var/name_override //For temporary visible name changes
 
 	var/heart_attack = 0
+
+	var/drunkenness = 0 //Overall drunkenness - check handle_alcohol() in life.dm for effects
+	var/drunk_state = NOT_DRUNK //A mob's state of drunkenness
+	var/alcohol_tolerance = AVERAGE_TOLERANCE //A mob's tolerance for alcohol poisoning
+	var/hungover = 0 //Whether or not they're recovering from heavy drinking and are still thirsty

```

Comment:
tee bee aych this should be on the reagents holder itself like addicton shit is

so monkeys can get hung over too